.github/workflows/TestBuild.yml

```diff
@@ -27,5 +27,7 @@ jobs:
         with:
           path: llvm
           key: build-${{ steps.llvm-build-dir.outputs.commit }}
+      - name: Build RVV enviroment
+        run: bash ./thirdparty/build-rvv-env.sh
```

Comment:
Looks like this script does not satisfy resources restrictions. Logs:
https://pipelines.actions.githubusercontent.com/serviceHosts/a2afae62-3cc1-4bba-8025-9c9e34c59e8a/_apis/pipelines/1/runs/286/signedlogcontent/2?urlExpires=2022-08-03T14%3A35%3A18.3447346Z&urlSigningMethod=HMACV1&urlSignature=wrCuK8O%2BbleXbepZnJqXJKvNXiS3HkcYJwARdsL0Dus%3D